src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Please don't disable linters for prop-types as requested by previous reviewer. PropTypes are very important to react components because they prevent to you from passing wrong props or wrong dataTypes. :blush: